.github/workflows/main.yml

```diff
@@ -26,6 +26,7 @@ jobs:
           arguments: build
 
       - name: Upload Build Artifacts
+        if: github.ref == 'refs/heads/develop'
```

Comment:
Just to be clear, with this workflow we are only publishing changes to develop or from develop?

Reply:
Good catch, I made this master